api/src/pcapi/routes/external/zendesk.py

```diff
@@ -13,7 +13,12 @@
 @public_api.route("/webhooks/zendesk/ticket_notification", methods=["POST"])
 @spectree_serialize(on_success_status=204, on_error_statuses=[400, 403])
 def zendesk_webhook_ticket_notification(body: zendesk_validation.WebhookRequest) -> None:
-    logger.info("Zendesk webhook called: %s", body, extra=body.dict())
+    rgpd_body = {"is_new_ticket": body.is_new_ticket, "requester_id": body.requester_id, "ticket_id": body.ticket_id}
+    logger.info(
+        "Zendesk webhook called: %s",
+        rgpd_body,
+        extra=rgpd_body,
+    )
```

Comment:
Les noms de variables sont généralement en anglais : `gdpr` et non `rgpd`.
Mais ce n'est pas clair, j'utiliserais plutôt le terme `anonymized`
```suggestion
    anonymized_body = {"is_new_ticket": body.is_new_ticket, "requester_id": body.requester_id, "ticket_id": body.ticket_id}
    logger.info("Zendesk webhook called: %s", anonymized_body, extra=anonymized_body)
```